jumpscale/sals/reservation_chatflow/reservation_chatflow.py

```diff
@@ -13,6 +13,10 @@
 import random
 import requests
 import time
+from redis import Redis
```

Comment:
I think no need for the native client unless we're making something custom, we are using the same database ?